webextension/scripts/background.js

```diff
@@ -391,12 +391,13 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
     var wayback_url = message.wayback_url
     var url = page_url.replace(/https:\/\/web\.archive\.org\/web\/(.+?)\//g, '')
     var open_url = wayback_url + encodeURI(url)
+    let isBulkSave = message.isBulkSave
```

Comment:
Please make it like `message.isBulkSave || false`

Reply:
I made the changes. Thank you!